bundle/next/eclipse-che-preview-openshift/manifests/che-operator.clusterserviceversion.yaml

```diff
@@ -75,12 +103,125 @@ metadata:
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/eclipse-che/che-operator
     support: Eclipse Foundation
-  name: eclipse-che-preview-openshift.v7.48.0-445.next
+  name: eclipse-che-preview-openshift.v7.48.0-549.next
   namespace: placeholder
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
+      - description: 'The `CheCluster` custom resource allows defining and managing
+          Eclipse Che server installation. Based on these settings, the  Operator
+          automatically creates and maintains several ConfigMaps: `che`, `plugin-registry`,
+          `devfile-registry` that will contain the appropriate environment variables
+          of the various components of the installation. These generated ConfigMaps
+          must NOT be updated manually.'
+        displayName: Eclipse Che instance Specification
+        kind: CheCluster
+        name: checlusters.org.eclipse.che
+        resources:
+          - kind: ClusterRole
+            name: ''
+            version: v1
+          - kind: ClusterRoleBinding
+            name: ''
+            version: v1
+          - kind: ConfigMap
+            name: ''
+            version: v1
+          - kind: Deployment
+            name: ''
+            version: apps/v1
+          - kind: Ingress
+            name: ''
+            version: v1
+          - kind: Role
+            name: ''
+            version: v1
+          - kind: RoleBinding
+            name: ''
+            version: v1
+          - kind: Route
+            name: ''
+            version: v1
+          - kind: Secret
+            name: ''
+            version: v1
+          - kind: Service
+            name: ''
+            version: v1
+        specDescriptors:
+          - description: Development environment default configuration options
+            displayName: Workspaces
+            path: workspaces
+          - description: Che operands configuration
```

Comment:
@tolusha @l0rd was thinking about the `operands` and technically not all the containg properties are operands e.g. `devWorkspace`, `imagePuller`, `extraProperties`, `metrics`
maybe it would be better to stick to nutaral `components` from the semantic perspective?

Reply:
`components` works for me

Reply:
Ok let's go with `components` then.

I got 2 feedbacks: 
1) `networking` would be better than `ingress`
2) `workspaces` is an overloaded term, maybe `devEnvironments` or `remoteEnvironments` would be better

What do you think?